api/v1/slurmcluster_types.go

```diff
@@ -68,6 +68,25 @@ type SlurmClusterSpec struct {
 	//
 	// +kubebuilder:validation:Optional
 	Telemetry *Telemetry `json:"telemetry,omitempty"`
+
+	// PartitionConfiguration define partition configuration of slurm worker nodes
+	// https://slurm.schedmd.com/slurm.conf.html#SECTION_PARTITION-CONFIGURATION
+	// +kubebuilder:validation:Optional
+	PartitionConfiguration PartitionConfiguration `json:"partitionConfiguration,omitempty"`
+}
+
+type PartitionConfiguration struct {
+	// ConfigType
+	// +kubebuilder:validation:Enum=default;custom
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="default"
+	ConfigType string `json:"configType,omitempty"`
+	// RawConfig define partition configuration as raw multi string started with PartitionName
+	// Example:
```

Comment:
This example is odd. Aren't PriorityTiers used for overlapping partitions?

Reply:
Ok, I changed it